src/test-utils/mocks.ts

```diff
@@ -388,3 +390,11 @@ export function createMockSubsidy(
 ): DeepMocked<Subsidy> {
   return createMock<Subsidy>(partial);
 }
+
+export function createMockResultSet<T extends any[]>(data: T): ResultSet<T> {
```

Comment:
I am not sure this would work:

```suggestion
export function createMockResultSet<T extends unknown[]>(data: T): ResultSet<T> {
```

otherwise add a comment to disable the `any` lint rule for this line